Exec/science/flame/analysis/flame_speed.py

```diff
@@ -17,7 +16,7 @@ class Profile:
     """read a plotfile using yt and store the 1d profile for T and enuc"""
 
     def __init__(self, plotfile):
-        ds = yt.load(plotfile)
+        ds = yt.load(plotfile, hint="castro")
```

Comment:
in yt, is there a change under the selection of hint?

Reply:
I am not sure what you are asking, but yt has trouble identifying a file as being from Castro instead of just BoxLib, so the hint tells it to try Castro.

Reply:
well, for a Castro pltxxxxx output (not a `profile.dat` output),  the code `yt.load()` doesn't seem to work, even with hint='castro'. For example, if I use `yt.load()` on the nova problem I get:
```
---------------------------------------------------------------------------
YTAmbiguousDataType                       Traceback (most recent call last)
/tmp/ipykernel_108688/1773296539.py in <module>
      1 # load data from file
----> 2 ds = yt.load('plt66500/')

~/.conda/envs/ytlook/lib/python3.7/site-packages/yt/loaders.py in load(fn, *args, **kwargs)
     94 
     95     if len(candidates) > 1:
---> 96         raise YTAmbiguousDataType(fn, candidates)
     97 
     98     raise YTUnidentifiedDataType(fn, *args, **kwargs)

YTAmbiguousDataType: Multiple data type candidates for plt66500
The following independent classes were detected as valid :
<class 'yt.frontends.boxlib.data_structures.CastroDataset'>
<class 'yt.frontends.boxlib.data_structures.AMReXDataset'>
A possible workaround is to directly instantiate one of the above.
Please report this to https://github.com/yt-project/yt/issues/new
``` 


A workaround I use is: `yt.CastroDataset()`.

Reply:
you need to make sure you are using yt 4.0 or later.

Reply:
yes, I am using 4.0.1. I am running it from nersc